packages/belos/src/BelosMinresIter.hpp

```diff
@@ -594,8 +602,8 @@ class MinresIter : virtual public MinresIteration<ScalarType,MV,OP> {
 
       // Update x:
       // x = x + phi*w;
-      //MVT::MvAddMv( one, *cur_soln_vec, phi, *W_, *cur_soln_vec );
-      lp_->updateSolution( W_, true, phi );
```

Comment:
I thought Christian's comment meant to just delete the comment on line 597??

Reply:
So, the MINRES solver updates the solution like the CG solver does, so the linear problem just needs to be aware of the fact that the solution has been updated manually.  I originally thought that it would be better to pass the update to the updateSolution call and let it handle unraveling the preconditioning, but that was wrong.  The code has been returned to the previous state.